### ISSUE TITLE
Fruit trees drop seeds

### ISSUE THREAD

**Issue by Balentay** [StardewModders/mod-ideas#220]
All other trees drop seeds, including mushroom trees.  It would be cool if fruit trees did too.

**Comment by KarBoy2314PL** [comment]
can't you do this with a seed maker already?

**Comment by minervamaga** [comment]
No, you can't.  There is a CFR machine for making saplings from fruit though.

**Comment by dittusch** [comment]
I'm currently developing such a mod.

**Comment by tiakall** [comment]
Vanilla trees now drop saplings when chopped in 1.6!
